voyager_archive/templates/voyager_archive/vendor_display.html

```diff
@@ -0,0 +1,121 @@
+{% extends 'voyager_archive/base.html' %}
+
+{% block content %}
+<form name="search_archive" id="search_archive" method="POST" enctype="multipart/form-data">
+    {% csrf_token %}
+    <table>
+        {{ form.as_table }}
+    </table>
+    <br>
+    <button type="submit">Search</button>
+</form>
+<hr/>
+
+
+<div class="row">
+    <div class="column">
+        <h1>Vendor Information</h1>
+        <table class="data-display">
+            <tr>
+                <td>Vendor Code</td>
+                <td>{{ vendor.vendor_code }}</td>
+            </tr>
+            <tr>
+                <td>Vendor Name</td>
+                <td>{{ vendor.vendor_name }}</td>
+            </tr>
+            <tr>
+                <td>Vendor ID</td>
+                <td>{{ vendor.vendor_id }}</td>
+            </tr>
+            <tr>
+                <td>Vendor Type</td>
+                <td>{{ vendor.vendor_type }}</td>
+            </tr>
+            <tr>
+                <td>Institution ID</td>
+                <td>{{ vendor.institution_id }}</td>
+            </tr>
+            <tr>
+                <td>Federal Tax ID</td>
+                <td>{{ vendor.federal_tax_id }}</td>
+            </tr>
+            <tr>
+                <td>Vendor Note</td>
+                <td>{{ vendor.vendor_note }}</td>
+            </tr>
+            <tr>
+                <td>Default Currency</td>
+                <td>{{ vendor.default_currency }}</td>
+            </tr>
+            <tr>
+                <td>Claim Interval</td>
+                <td>{{ vendor.claim_interval }}</td>
+            </tr>
+            <tr>
+                <td>Claim Count</td>
+                <td>{{ vendor.claim_count }}</td>
+            </tr>
+            <tr>
+                <td>Cancel Interval</td>
+                <td>{{ vendor.cancel_interval }}</td>
+            </tr>
+            <tr>
+                <td>Creation Date</td>
+                <td>{{ vendor.create_date }}</td>
+            </tr>
+            <tr>
+                <td>Creation Operator ID</td>
+                <td>{{ vendor.create_operator_id }}</td>
+            </tr>
+            <tr>
+                <td>Modification Date</td>
+                <td>{{ vendor.modify_date }}</td>
+            </tr>
+            <tr>
+                <td>Modification Operator ID</td>
+                <td>{{ vendor.modify_operator_id }}</td>
+            </tr>
+            <tr>
+                <td>Number of Associated Accounts</td>
+                <td>{{ vendor_accts.count }}</td>
+            </tr>
+        </table>
+    </div>
+
+    <div class="column">
+        {% if vendor_accts %}
+        <h1>Vendor Account Information</h1>
+        {% for acct in vendor_accts %}
```

Comment:
Please add something to help visually separate multiple accounts.  I should have provided multiple sample accounts (👎 me...)

I modified my copy of `sql_support/sample_data.sql`, copying the 1 row and tweaking the `account_id`, `account_number` and `account_name`:
![vendor_account_samples](https://user-images.githubusercontent.com/2213836/194424841-0a90908e-d58a-48c7-8318-2bfd49f89cbb.png)

The resulting display is OK, with a separate table for each account, but the visual separation is subtle:
![vendor_account_multiple](https://user-images.githubusercontent.com/2213836/194424999-ae91768b-da5f-46c2-86dd-a31b0c6c2fd2.png)